Design.md

```diff
@@ -0,0 +1,187 @@
+---
+authors: Brian Green (bgreen007@cox.net)
+state: draft
+---
+
+## What
+Job worker service that provides an API to run arbitrary Linux processes.
+
+## Details
+
+
+### Security
+Use mTLS for gRPC communication.
```

Comment:
What versions of TLS will be supported ?

Reply:
1.3. added.